dll/win32/msi/dialog.c

```diff
@@ -3160,7 +3160,14 @@ static void msi_dialog_vcl_add_columns( msi_dialog *dialog, msi_control *control
          */
         if ( !wcsncmp( num, L"-", 1 ) || !str_is_number( num ) ) {
             msi_free( num );
+/* Temporary workaround for CORE-18749
+ * new issue will be opened for improvement on jira
```

Comment:
This line does not seem wanted in code.
Moreover, you have still not explained why this "workaround", as is, is not a regression with regard to existing feature: what happens if `num[0] == '-'`, for example?